bundles/AdminBundle/public/js/pimcore/settings/docTypes.js

```diff
@@ -149,13 +149,15 @@ pimcore.settings.document.doctypes = Class.create({
             },
             {
                 text: t("priority"),
-                flex: 50,
+                flex: 30,
                 sortable: true,
                 dataIndex: 'priority',
-                editor: new Ext.form.ComboBox({
-                    store: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10],
+                editor: new Ext.form.NumberField({
+                    allowBlank: true,
```

Comment:
If we allow "blank" or null here we have to change setPriority method to allow null. We should then also remove the int cast. If we do not allow null values we could still remove the int cast though :D 
Currently it looks like the following:
```php
   public function setPriority(int $priority): static
    {
        $this->priority = (int) $priority;

        return $this;
    }
```